packages/integration-tests/src/helper.ts

```diff
@@ -0,0 +1,89 @@
+import { execSync } from "child_process";
+import { ethers } from "ethers";
+import { promises as fs } from "fs";
+import * as path from "path";
+import * as request from "supertest";
+import { Provider } from "zksync-web3";
+
+import { environment, localConfig } from "./config";
+import { Logger } from "./entities";
+
+import type { BaseProvider } from "@ethersproject/providers/src.ts/base-provider";
+
+export class Helper {
+  async txHashLogger(txType: string, txValue: string, tokenName?: string) {
+    const logMessage = `TxHash for ${txType} ${Logger.textSeparator} ${txValue}`;
+
+    if (tokenName === undefined) {
+      return console.log(logMessage);
+    } else {
+      return console.log(logMessage, ` ${tokenName}`);
+    }
+  }
+
+  async executeScript(script: string) {
+    const output = execSync(script, { encoding: "utf-8" });
+
+    try {
+      console.log(`> Run NPM Script "${script}":\n`, output);
+      return output;
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  async getStringFromFile(fileName: string) {
+    const absoluteRoute = path.join(__dirname, "..", fileName);
+
+    try {
+      return await fs.readFile(absoluteRoute, { encoding: "utf-8" });
+    } catch {
+      console.log(`There is no the expected file: ${fileName}`);
```

Comment:
Do you want it to return undefined in case there is no file? It doesn't look like, better just let it fail as I assume you always expect the file to exist, so if it doesn't let it fail.

Reply:
updated